scripts/us_census/pep/monthly_population_estimate/test_data/expected_USA_Population_Count.csv

```diff
@@ -1,25 +1,25 @@
-﻿Date,Location,Count_Person_InUSArmedForcesOverseas,Count_Person_USResident,Count_Person_USResidentOrInUSArmedForcesOverseas,Count_Person_Civilian,Count_Person_Civilian_NonInstitutionalized
-2002-12,country/USA,308953,288910708,289219661,287657759,283567193
-2002-11,country/USA,299808,288716345,289016153,287455156,283367384
-2002-10,country/USA,298850,288484263,288783113,287215377,283130237
-2002-09,country/USA,300697,288233967,288534664,286963476,282881204
-2002-08,country/USA,305751,287973348,288279099,286707011,282627594
-2002-07,country/USA,300904,287726647,288027551,286460080,282383435
-2002-06,country/USA,295655,287493821,287789476,286230294,282154728
-2002-05,country/USA,301620,287250118,287551738,285998163,281923687
-2002-04,country/USA,302569,287025879,287328448,285783873,281710480
-2002-03,country/USA,306955,286817724,287124679,285584303,281512166
-2002-02,country/USA,296700,286634747,286931447,285395443,281324390
-2002-01,country/USA,298260,286429775,286728035,285203166,281132861
-2001-12,country/USA,287595,286225459,286513054,284988967,280920258
-2001-11,country/USA,283100,286004703,286287803,284777261,280709686
-2001-10,country/USA,277453,285770015,286047468,284562038,280495455
-2001-09,country/USA,223757,285571270,285795027,284324928,280259557
-2001-08,country/USA,227016,285298620,285525636,284058779,279994583
-2001-07,country/USA,227463,285039803,285267266,283803584,279740517
-2001-06,country/USA,227626,284795649,285023275,283566906,279505761
-2001-05,country/USA,225333,284550030,284775363,283321122,279261915
-2001-04,country/USA,222221,284327688,284549909,283092935,279035703
-2001-03,country/USA,221303,284100861,284322164,282863438,278808323
-2001-02,country/USA,221982,283890737,284112719,282654413,278601238
-2001-01,country/USA,208561,283690962,283899523,282442439,278390935
\ No newline at end of file
+Date,Location,Count_Person_InUSArmedForcesOverseas,Count_Person_USResident,Count_Person_USResidentOrInUSArmedForcesOverseas,Count_Person_Civilian,Count_Person_Civilian_NonInstitutionalized
+2002-12,country/USA,308953,288910708.0,289219661.0,287657759.0,283567193.0
```

Comment:
Column 3 is a derived column in which we are computing using
df['Count_Person_InUSArmedForcesOverseas'] = df['Resident Population Plus Armed Forces Overseas'].astype('int') - df['Resident Population'].astype('int'). Thats is why it is left as int. 
However it can be changed if needed as well.

Reply:
Changed values to make everything as int.